### ISSUE TITLE
Unable to download addon for nvda 2019.3

### ISSUE THREAD

**Issue by shm-vadim** [jscholes/nvda-speech-history#6]
Hello! In readme written, that the addon supports nvda 2019.3. But I download it by clicking stable version link and can not install. Because nvda shows an error. Could you create a new release for nvda 2019.3 and update the link for stable version of the addon?
